tools/update-base64.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+set -e
+# Shell script to update base64 in the source tree to a specific version
+
+BASE_DIR="$( pwd )"
```

Comment:
```suggestion
BASE_DIR=$(cd "$(dirname "$0")/.." && pwd)
```

This should make it work from anywhere.

Reply:
Fixed!